Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -67,7 +72,7 @@ final class EvaluationInteractorImpl: EvaluationInteractor {
                         try evaluationDao.deleteAllAndInsert(userId: user.id, evaluations: newEvaluations)
                     } catch let error {
                         logger?.error(error)
-                        completion?(result)
+                        completion?(.failure(error: .init(error: error), featureTag: featureTag))
```

Comment:
You changed here to return also the feature tag so that it can be set in the metrics event?
If so, we should implement a unit test checking the completion values matches the values set in the metrics event.

Because the unit test passed, but it wasn't set the feature tag correctly.

Reply:
Added https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/894812a76104edcc1b1a3a83b0095d5735691ed2